nextflow.config

```diff
@@ -0,0 +1,31 @@
+manifest {
+    name = 'ERVmap'
+    mainScript = 'main.nf'
+}
+
+docker {
+    enabled = true
+    temp = 'auto'
+    runOptions = '--rm -u $(id -u):$(id -g)'
+}
+
+params {
+    inputDir=''
+    outPrefix='results/test.'
+    cpus=1
+    limitMemory=30000000
+    debug='off'
+}
+
+process {
+    withName: STARAlignment {
```

Comment:
Should the name be ERValign?

Reply:
Yes, I've changed the actual configuration files, but I forgot to change the template.